src/components/layout/NavBar.vue

```diff
@@ -1,10 +1,17 @@
 <template>
   <header class="header" id="header">
     <nav class="inner">
-       <div @click="resetApp" class="site-name site-logo">
-        <img class="logo-img" src="../../assets/sedaily-logo.png">
-          Software Daily
-       </div>
+      <div class="navbar-centered">
+        <div @click="resetApp" class="navbar-centered">
+          <img @click="resetApp" class="logo-img site-logo" src="../../assets/sedaily-logo.png">
+          <span @click="resetApp" class="site-name">Software Daily</span>
```

Comment:
Do we need `resetApp` on all three of these elements? We should only need it on the parent element. If the children are preventing the action for some reason we can use css and just add `pointer-events: none;` to prevent them from getting in the way.

Reply:
I noticed the method `resetApp` reloads the page. Do we still need that if we are clearing the `searchTerm` and setting the path to `/`?

Reply:
I had noticed this too but was afraid to change for some other reason. But I will remove that

Reply:
```router.push('/location').catch(err => {})``` this catch is really needed for duplicated navigation error. That's really odd 
https://github.com/vuejs/vue-router/issues/2872